src/app/page.tsx

```diff
@@ -1,11 +1,20 @@
 "use client";
 import Image from "next/image";
+import imageLogoMarquei from "../app/LOGO_MARQUEI.jpeg";
 import resourcePlaceholder from "./assets/resource-placeholder.png";
 import "./style.css";
 
 export default function Home() {
   return (
     <main>
+      <div>
+        <header>
+          <Image src={imageLogoMarquei} alt="The logo Marquei" />
+        </header>
+
+        <span className="show">Nome</span>
+      </div>
```

Comment:
Vou sugerir aqui algumas alterações pra dar uma melhorada na hierarquia do conteúdo aqui.

```suggestion
      <header className="page-header">
        <div className="header-image-container">
          <Image src={imageLogoMarquei} alt="The logo Marquei" />
        </div>

        <span className="header-user-tag">Nome</span>
      </div>
```